lib/screens/VeggieDetail.dart

```diff
@@ -57,34 +57,92 @@ class _VeggieDetailState extends State<VeggieDetail> {
     });
   }
 
+  Widget _regularComponent(Widget servingInfo) {
```

Comment:
I pulled this out because inside of the build method it was super indented and hard to read

Reply:
Can this be an instance where we might want to refactor this widget as a separate widget instead of a method?